docs/package.json

```diff
@@ -44,7 +44,9 @@
   },
   "devDependencies": {
     "@docusaurus/module-type-aliases": "^2.0.0-beta.3",
+    "@docusaurus/types": "^2.0.0-beta.18",
```

Comment:
Can we not pull in a version that matches our other docusaurus deps?

Reply:
Done.